l3build-aux.lua

```diff
@@ -82,11 +88,8 @@ end
 ---`texlua l3build.lua` -> `/Library/TeX/texbin/l3build.lua` or `./l3build.lua`
 ---@return string
 local function get_script_name()
-  if match(arg[0], "l3build$") or match(arg[0], "l3build%.lua$") then
-    return lookup("l3build.lua")
-  else
-    return arg[0] -- Why no lookup here?
-  end
+  local dir, base = splitpath(l3build.PATH)
+  return abspath(dir) .. '/' .. base
```

Comment:
`l3build.PATH` is `lookup("l3build.lua")` when `l3build` calling sequence is standard.